submissions/Serge044/Classic-Frogger-Game/app.js

```diff
@@ -0,0 +1,151 @@
+const Enemy = function (x, y) {
+  this.width = 75;
+  this.x = x;
+  this.y = y;
+  this.border = { left: -100, right: 505 };
```

Comment:
Yet another number obviously connected with `stepX` (or whatever it actually is).